tests/sur/quota.js

```diff
@@ -339,8 +395,52 @@ function multiObjectDelete(bucket, keys, size, callback) {
         mockScuba = scuba;
 
         before(done => {
-            const config = getConfig('default', { signatureVersion: 'v4', maxRetries: 0 });
-            s3Client = new S3(config);
+            const config = getConfig('default', { 
+                maxRetries: 0,
+            });
+
+            s3Client = new S3Client({
+                ...config,
+                // Disable ALL automatic checksum handling
+                requestChecksumCalculation: 'WHEN_REQUIRED',
+                responseChecksumValidation: 'WHEN_REQUIRED',
+                checksumDisabled: true,
+                disableRequestCompression: true,
+                // Force the client to not add automatic headers
+                useGlobalEndpoint: false,
+            });
+            
+            // Add middleware to strip ALL checksum headers
+            s3Client.middlewareStack.add(
```

Comment:
Can we maybe add a comment to explain why we need this? And maybe a ticket number if that could be removed, so we just support the standard headers